app/charts/map/map.tsx

```diff
@@ -219,7 +219,12 @@ export const MapComponent = () => {
               controller={{ type: MapController }}
               getCursor={() => "default"}
             >
-              {showBaseLayer && <StaticMap mapStyle={baseLayerStyle} />}
+              {showBaseLayer && (
+                <StaticMap
+                  mapStyle={baseLayerStyle}
+                  attributionControl={false}
```

Comment:
I made an assumption that we don't have to display Swisstopo's logo at the bottom of a map, at least this is how it was in a prototype. I think it makes sense to ask this question during a meeting with Swisstopo, just to be sure.

Reply:
> Terms of use: the map combines different data sets with different terms of use. The map and the geoservice that provides it may be used in accordance with the [General Terms of Use and Operating Conditions of the Federal Spatial Data Infrastructure FSDI](http://www.geo.admin.ch/terms-of-use). In the case of digital or analog representations and publications by the administrative units of the Federal Administration, the following source reference must be provided in all cases: (with appropriate links) [© swisstopo](https://www.swisstopo.ch/),[© MapTiler](https://www.maptiler.com/copyright/) [© OpenStreetMap contributors](https://www.openstreetmap.org/copyright)

https://www.geo.admin.ch/en/geo-services/geo-services/portrayal-services-web-mapping/vector_tiles_service.html

I think here the problem is that the attribution control is not displayed properly. I think we are lacking CSS classes for attribution elements like `.mapboxgl-ctrl mapboxgl-ctrl-attrib .mapboxgl-compact` as I see that in the above documentation, the "i" is displayed properly (and on hover it shows swisstopo). I think we may have to copy some CSS classes into our global CSS.

Reply:
Let's tackle this in another issue right now, disabling attribution for now, issue created #369